### ISSUE TITLE
Test Failures

### ISSUE THREAD

**Issue by dangulko** [angular/protractor#273]
During one of my test runs I had a failure in which a test did not get a return of an element which it was looking for. The issue was that after the initial failure all subsequent tests did not seem to run at all and therefore failed. The subsequent test cases were separate tests and not part of the initial failing test. Is this normal or should the separate tests run despite the previous test failure? Unfortunately I don't have an example output as this this was seen only once and not repeatable after that. I do remember seeing some error thrown about jasmine promise. Any feedback is helpful.
